tools/ckb4ibc-test/src/tests/packet.rs

```diff
@@ -0,0 +1,188 @@
+}
+
+fn error_cast<T: ToString>(error: T) -> eyre::Error {
+    eyre!("{}", error.to_string())
+}
+
+fn generate_ckb_sdk_config(
+    config: &Config,
+    chain: &impl ChainHandle,
+    channel_id: &ChannelId,
+) -> EyreResult<(SdkConfig, String)> {
+    let chain_id = chain.id();
+    let generic_config = config
+        .find_chain(&chain_id)
+        .ok_or(eyre!("{chain_id} not found"))?;
+    let ckb4ibc_config = if let ChainConfig::Ckb4Ibc(config) = generic_config {
+        config
+    } else {
+        return Err(eyre!("{chain_id} is not ckb4ibc type"));
+    };
+    let relayer_address = {
+        let key_base: KeyRing<Secp256k1KeyPair> =
+            KeyRing::new(Default::default(), "ckb", &chain_id)?;
+        let relayer_pubkey = key_base.get_key(&ckb4ibc_config.key_name)?.public_key;
+        let payload = AddressPayload::from_pubkey(&relayer_pubkey);
+        Address::new(NetworkType::Testnet, payload, true)
+    };
+    let metadata_type_args = ckb4ibc_config.lc_client_type_args(5u64.try_into()?)?;
+    let metadata_script = Script::new_builder()
+        .code_hash(TYPE_ID_CODE_HASH.pack())
+        .hash_type(ScriptHashType::Type.into())
+        .args(metadata_type_args.to_vec().pack())
+        .build();
+    let channel_number = u16::from_str(
+        channel_id
+            .to_string()
+            .split('-')
+            .last()
+            .expect("channel id"),
+    )?;
+    let sdk_config = SdkConfig {
+        user_lock_script: AddressOrScript::Address(AddressString(relayer_address)),
+        axon_metadata_type_script: AddressOrScript::Script(metadata_script.into()),
+        channel_contract_type_id_args: ckb4ibc_config.channel_type_args.clone(),
+        packet_contract_type_id_args: ckb4ibc_config.packet_type_args.clone(),
+        channel_id: channel_number,
+        confirmations: 1,
+    };
+    Ok((sdk_config, ckb4ibc_config.ckb_rpc.to_string()))
+}
+
+fn generate_send_packet_transaction(
+    rt: &Runtime,
+    sdk_config: &SdkConfig,
+    ckb_rpc: &SdkRpcClient,
+    target_port_id: String,
+    target_channel_id: String,
+    payload: Vec<u8>,
+) -> EyreResult<TransactionView> {
+    let metadata_celldep = rt
+        .block_on(get_axon_metadata_cell_dep(ckb_rpc, sdk_config))
+        .map_err(error_cast)?;
+    let channel_contract_celldep = rt
+        .block_on(get_channel_contract_cell_dep(ckb_rpc, sdk_config))
+        .map_err(error_cast)?;
+    let ibc_channel = rt
+        .block_on(IbcChannelCell::get_latest(ckb_rpc, sdk_config))
+        .map_err(error_cast)?;
+    let send_packet = IbcPacket {
+        packet: Packet {
+            destination_port_id: target_port_id,
+            destination_channel_id: target_channel_id,
+            data: payload,
+            ..Default::default()
+        },
+        tx_hash: None,
+        status: PacketStatus::Send,
+    };
+    let (send_packet_tx, _) = assemble_send_packet_partial_transaction(
```

Comment:
@blckngm this function will emit `get channel cell` error, please have a check on it, the detail is in the GitHub CI output https://github.com/synapseweb3/forcerelay/actions/runs/5939949923/job/16107478392?pr=305

Reply:
Use `{:#}` or Debug to get a more detailed error message:

https://docs.rs/anyhow/latest/anyhow/struct.Error.html#display-representations

Reply:
Do you think there's anything wrong here:

https://github.com/synapseweb3/forcerelay-ckb-sdk/blob/1737ef35c38dea25dfd411567ef0ae6f956e06f4/src/config.rs#L59-L75

Get latest channel cell just searches for the last cell with this lock.